addons/web/static/src/views/fields/many2many_tags_avatar/many2many_tags_avatar_field.js

```diff
@@ -39,32 +37,46 @@ export const many2ManyTagsAvatarField = {
 
 registry.category("fields").add("many2many_tags_avatar", many2ManyTagsAvatarField);
 
-export class ListKanbanMany2ManyTagsAvatarField extends Many2ManyTagsAvatarField {
+export class ListMany2ManyTagsAvatarField extends Many2ManyTagsAvatarField {
     static props = {
         ...Many2ManyTagsAvatarField.props,
         itemsVisible: { type: Number, optional: true },
     };
     static defaultProps = {
         ...Many2ManyTagsAvatarField.defaultProps,
-        itemsVisible: 3,
+        itemsVisible: 5,
     };
+}
 
-    getTagProps(record) {
-        return {
-            ...super.getTagProps(record),
-            img: `/web/image/${this.relation}/${record.resId}/avatar_128`,
-        };
-    }
+export const listMany2ManyTagsAvatarField = {
+    ...many2ManyTagsAvatarField,
+    component: ListMany2ManyTagsAvatarField,
+    extractProps: (fieldInfo) => ({
+        ...many2ManyTagsAvatarField.extractProps(fieldInfo),
+        itemsVisible: 5,
+    }),
+};
+
+registry.category("fields").add("list.many2many_tags_avatar", listMany2ManyTagsAvatarField);
+
+export class KanbanMany2ManyTagsAvatarField extends Many2ManyTagsAvatarField {
+    static props = {
+        ...Many2ManyTagsAvatarField.props,
+        itemsVisible: { type: Number, optional: true },
+    };
+    static defaultProps = {
+        ...Many2ManyTagsAvatarField.defaultProps,
+        itemsVisible: 3,
+    };
 }
 
-export const listKanbanMany2ManyTagsAvatarField = {
+export const kanbanMany2ManyTagsAvatarField = {
     ...many2ManyTagsAvatarField,
-    component: ListKanbanMany2ManyTagsAvatarField,
+    component: KanbanMany2ManyTagsAvatarField,
     extractProps: (fieldInfo) => ({
         ...many2ManyTagsAvatarField.extractProps(fieldInfo),
-        itemsVisible: fieldInfo.viewType === "list" ? 5 : 3,
+        itemsVisible: 3,
```

Comment:
same here.

Actually, does it even need to be a props? It can be directly set in each Component.